easybuild/easyblocks/t/trinity.py

```diff
@@ -54,6 +54,7 @@ def __init__(self, *args, **kwargs):
         EasyBlock.__init__(self, *args, **kwargs)
 
         self.build_in_installdir = True
+        self.path = ''
```

Comment:
ok, fine, keep it `''` then, but change the name to something less generic as suggested above